lib/bundles/inspec-compliance/target.rb

```diff
@@ -46,10 +46,7 @@ def get_profile_name(uri)
     def build_target_url(target)
       owner, profile = target.split('/')
       config = Compliance::Configuration.new
-      url = "#{config['server']}/owners/%owner_name%/compliance/%profile_name%/tar"
-            .gsub('%owner_name%', owner)
-            .gsub('%profile_name%', profile)
-      url
+      "#{config['server']}/owners/#{owner}/compliance/#{profile}/tar"
```

Comment:
Thank you @srenatus for this hint: https://github.com/chef/inspec/pull/458/files#diff-e975c4d3a237ff3f9a0f22cc54abab4aR49